### ISSUE TITLE
Have a dedicated High Contrast theme or just leaving dark.css for high contrast mode

### ISSUE THREAD

**Issue by microdee** [gitextensions/gitextensions#10570]
### Feature description

I use Windows in high contrast mode, and the dark mode of GitExtensions 3 was looking OK with it. However I understand that in GitExtensions 4.0+ the theming support could no longer go on. However that made the UI look like this in high contrast mode:

![image](https://user-images.githubusercontent.com/1009962/209236414-3561f8d0-45e1-4402-92aa-fa9a5a64e7f7.png)

In 4.0.0 where the dark theme was still selectable, it could still restore themed parts of the app to a readable state, like so:

![image](https://user-images.githubusercontent.com/1009962/209237264-4d6680ad-0fb4-4016-92f7-444fb1d1bed7.png)

With the latest update "dark.css" has been removed, and I had to restore it manually to make GitExtensions usable again in high contrast.

Here's the feature request part: would it be possible to still include the dark.css for the case if the user has high contrast mode enabled? Maybe only let them select it if HC mode is detected (somehow), in case the original intent of removing it was avoiding confusion.

### Environment

- Git Extensions 4.0.1.15887
- Build f2567dea28bbf0e2cbcd2a95d4c7200e47d0d671
- Git 2.38.1.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.12
- DPI 96dpi (no scaling)
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 3.1.22 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 3.1.32 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 5.0.13 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 5.0.17 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 6.0.10 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
    Microsoft.WindowsDesktop.App 6.0.12 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

**Comment by gerhardol** [comment]
Maybe add a highcontrast_dark.css?
Then it is not likely to be unkowingly selected.

Any tuning that should be done compared to current dark.css?

**Comment by microdee** [comment]
yeah I guess I can create a high contrast version of dark.css, I just wanted to raise awareness to it. if we want to minimize work the vanilla dark.css looks ok as-is, no tuning is needed for functionality there

**Comment by gerhardol** [comment]
Closed in #10621
